lib/puppet/functions/lookup.rb

```diff
@@ -0,0 +1,80 @@
+# @see lib/puppet/parser/functions/lookup.rb for documentation
```

Comment:
Cannot be done this way - the documentation should be here and use yard. Unfortunately for the time being we need to duplicate the effort. The strings tool will be able to find this documentation. As soon as our docs pipeline is modified to use strings we will drop the 3x stub with old rdoc content.

For this purpose, other 4x functions have not yet fully exploited the use of yarddoc tags (to make the maintenance of the documentation just a copy/paste for the time being.